vault/changelog.early-seed.md

```diff
@@ -2,29 +2,29 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1647629183354
+updated: 1649437435333
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.89.0
+## 0.90.0
 
-Dendron 0.89 has sprouted 🌱
+Dendron 0.90 has sprouted 🌱
 
-Interested in contributing to the development of Dendron? We've added a [CONTRIBUTING.md](https://github.com/dendronhq/dendron/blob/master/CONTRIBUTING.md) file so that you can find out how to get started shaping the future of Dendron!
+**Vault-Level URLs**: Vaults can now be configured with their own URL. These URLs will be used when copying, exporting or publishing notes, replacing wikilinks with the correct URL for the vault.
```

Comment:
they will be used when copying/exporting - publishing is still set by dendron.yml